news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+Not at all, I'm pretty sure I was bootynator for about 2 days until having a shower thought that "xooty" would look much cooler than "booty" 😼
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Speaking of nicknames, what made you change your username to "his kitten" for some time?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+That was from my first subathon in 2023! I had a subscriber goal to change my name to anything chat chose, and they ended up choosing "his kitten". Everyone wanted me to change it back to xootynator after a while but I maybe got a little attached to the name... meow :3
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+I remember you mentioning it in one of your previous interviews, but can you tell us what's the origin of SFA (Stan Full Area)? And how did that become your thing?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+Stan Full Area has a pretty funny origin, since it was actually out team name for the 2020 iteration of [MRC (Mahloola Regional Cup)](https://osu.ppy.sh/community/forums/topics/1136787). The team was me, [Rupertion](https://osu.ppy.sh/users/5774823) and [Eddie-](https://osu.ppy.sh/users/3898396), three players who played with the entire area of our tablets. No changes to the size of the active area at all. Eddie is a big kpop fan, so gave us the name Stan Full Area. Since then it's sort of just stuck as a tag to recognize other people with the same playstyle!
```

Comment:
```suggestion
Stan Full Area has a pretty funny origin, since it was actually out team name for the 2020 iteration of [MRC (Mahloola Regional Cup)](https://osu.ppy.sh/community/forums/topics/1136787). The team was me, [Rupertion](https://osu.ppy.sh/users/5774823) and [Eddie-](https://osu.ppy.sh/users/3898396), three players who played with the entire area of our tablets. No changes to the size of the active area at all. Eddie is a big k-pop fan, so gave us the name Stan Full Area. Since then, it's sort of just stuck as a tag to recognize other people with the same play style!
```